examples/toolchains/cc/README.md

```diff
@@ -0,0 +1,17 @@
+C++ Toolchain Example
+=====================
+
+This is an example C++ project that uses `rules_cc`. It should build and run both with or without Nix installed.
```

Comment:
Please clarify that this demonstrates two different modes. Currently, it could be misread that without Nix installed it would somehow install Nix automatically. This is of course not the case, without Nix installed this will fetch toolchains from the environment and be not as hermetic as the Nix setup.

Same for the other `README` files.